### ISSUE TITLE
Recipe conflict with default

### ISSUE THREAD

**Issue by fireglow** [adrido/darkage#2]
There is a conflict with the default game:
in init.lua:337
```
minetest.register_craft({
        output = "default:desert_stone 2",
        recipe = {
                {"default:desert_sand", "default:desert_sand"},
                {"default:desert_sand", "default:desert_sand"},
        }
})
```
That recipe gives default:desert_sandstone in the default game.

**Comment by adrido** [comment]
Thanks for reporting the bug. Ill fix it this week. 
Do you already have a idea for a alternative reciepe?

**Comment by fireglow** [comment]
On my server, I just removed the recipe (😿) and let default handle it.

**Comment by adrido** [comment]
Solved by 686d5316afa650602c85cc4c197a3cb866a82b9c
